src/report.js

```diff
@@ -212,6 +212,7 @@ class Report {
 
     if (this.report.coldstart) {
       this.report.labels.add('@iopipe/coldstart');
+      this.report.duration = Math.ceil(process.uptime() * 1e9);
```

Comment:
@ewindisch Isn't this the uptime since the process was created? Wouldn't that encompass many invocations?